src/libraries/Microsoft.PowerFx.Interpreter/Functions/IAsyncTexlFunction2.cs

```diff
@@ -12,5 +12,7 @@ namespace Microsoft.PowerFx.Core.Functions
     internal interface IAsyncTexlFunction2
     {
         Task<FormulaValue> InvokeAsync(FormattingInfo context, FormulaValue[] args, CancellationToken cancellationToken);
+
+        Task<FormulaValue> InvokeAsync(string url, CancellationToken cancellationToken);
```

Comment:
>InvokeAsync [](http://example.com/codeflow?start=27&length=11)

this should probably be on its own interface.

Reply:
Will do IAsyncTexlFunction3 : IAsyncTexlFunction2 with it

Reply:
We could give it a more specific name - this is just for URLs and pageable entries?

Reply:
actually - why does this even need to be on an interface at all? If the only caller is already in ServiceFunction, it doesn't need to be on an interface.

Reply:
Resolved offline
Removed the API from interface